### PR TITLE
fix: Add link to ENVIRONMENT_VARIABLES.md file

### LINKED ISSUE #210: Change the link for setting environmental variables
### Describe the bug
The present link for setting the environment variable is not clear enough.

### To Reproduce
1. Go to 'README.md'
2. Click on 'BIT Development Environment Setup Instruction '
3. Scroll down to '7th, Create the .env file using .env.template'
4. See error by clicking on 'environment variables - tba:'

### Expected behavior
The link should be replaced with [this](https://github.com/anitab-org/bridge-in-tech-backend/blob/develop/docs/ENVIRONMENT_VARIABLES.md)

### Screenshots
Screenshot from 2021-03-09 11-07-37

### Desktop (please complete the following information):
 - OS: [Ubuntu 18.04]
 - Browser [chrome]

### PR DESCRIPTION
**Pull request by HeroicHitesh**
### Description

Add a link to the `ENVIRONMENT_VARIABLES.md` file in the `ENV_SETUP_INSTRUCTION.md` file

Fixes #210 

### Type of Change:

- Documentation

### Checklist:

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [x] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

File: .github/ENV_SETUP_INSTRUCTION.md [hunk 1]
```diff
@@ -97,7 +97,7 @@ Now when you run the application using `python run.py` (after first completing s
 <img width="647" alt="Screen Shot 2020-07-15 at 5 39 46 pm" src="https://user-images.githubusercontent.com/29667122/87517460-38f8b580-c6c2-11ea-9bfb-a0117f0ee848.png">
 
 ## 7th, Create the `.env` file using `.env.template`
-Update the values of corresponding environment variables or make sure you exported the following [environment variables - tba]():
+Update the values of corresponding environment variables or make sure you exported the following [environment variables - tba](https://github.com/anitab-org/bridge-in-tech-backend/blob/develop/docs/ENVIRONMENT_VARIABLES.md):
```
Comment (mtreacy002) [inline .github/ENV_SETUP_INSTRUCTION.md]: > [environment variables - tba]

@HeroicHitesh , you can remove the "-tba` from the link name 😉

**Review by mtreacy002 — CHANGES_REQUESTED** [review]
@HeroicHitesh , well done for submitting your 1st PR. Just want to request one minor change. Once you've done this, I'll approve.

File: .github/ENV_SETUP_INSTRUCTION.md (hunk 1 above)
Comment (HeroicHitesh) [reply .github/ENV_SETUP_INSTRUCTION.md]: @mtreacy002 Sure. I was actually unsure what it means so I just left it unchanged.

**Review by epicadk — APPROVED** [review]
💯

**Review by mtreacy002 — APPROVED** [review]
Well done, @HeroicHitesh for the fix 🎉🎉🎉. Thank you again for contributing to the project. I'll approve this now. No testing needed as this is only affecting documentation.

**Pull request merged**